gax-java/gax-grpc/src/test/resources/README.md

```diff
@@ -0,0 +1,27 @@
+# Generating certificates and keys for testing mTLS-S2A
```

Comment:
IIRC, you mentioned to be that these were from grpc-java and are dummy certs, right? Can you update that in a small section here?

Reply:
nit: perhaps for future references, rephrase as guide to re-generate these test certs/keys?

Reply:
Yes, these are the same ones I added to s2a client in grpc-java. Updated in https://github.com/googleapis/sdk-platform-java/pull/3326/commits/2a50985211d44ca87423b580c617c5b1e92e3d22